.github/workflows/dev_builds.yml

```
@@ -5,6 +5,7 @@ on:
     branches:
       - main
       - 'dev-build/*'
+      - drosiek-windows-container
```

```suggestion
      - drosiek-windows-container
```

to Rremove before merge